src/index.html

```diff
@@ -4,14 +4,185 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    name="test-form"
+    >
+      <fieldset class="inputGroup">
+        <legend>Personal information</legend>
+        <div class="inputWrapper">
+          <label>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="inputWrapper">
+          <label>
+            Name:
+            <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
            <input
              type="text"
              name="surname"
              autocomplete="off"
            >
          </label>
        </div>
        <div class="inputWrapper">
          <label>
            Name:
            <input
              type="text"
              name="name"
              autocomplete="off"
            >
```
fix code style all over the page